third_party/terraform/utils/config.go.erb

```diff
@@ -2,411 +2,427 @@
 package google
```

Comment:
Looks like the tabs in this file were replaced with spaces; do you mind changing it back?